examples/angular/src/app/components/blogs/blogs.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, inject, signal } from '@angular/core';
+import { DOTCMS_CLIENT_TOKEN } from '../../client-token/dotcms-client';
+import { GenericContentlet } from '../../utils';
+import { ContentletsComponent } from '../contentlets/contentlets.component';
+import { Contentlet } from '@dotcms/client/src/lib/client/content/shared/types';
+
+@Component({
+  selector: 'app-blogs',
+  standalone: true,
+  imports: [ContentletsComponent],
+  template: ` <div class="flex flex-col">
+    <h2 class="text-2xl font-bold mb-7 text-black">Latest Blog Posts</h2>
+    @if(!!blogs().length) { <app-contentlets [contentlets]="blogs()" /> }
```

Comment:
@zJaaal  I think the format with prettier 3 was not applied

Reply:
Yep this file does not contain any prettier config, because it is an example for customers.

Reply:
But @zJaaal the code for customers should also have a good format, or not?

Reply:
We discussed at the start that we don't want to apply any prettier/eslint rules to code that is not going to be use by us. The customers can manage their own standards. 

@fmontes any thoughts?

Reply:
Whaterer Angular gives out of the box is fine, I don't want to add anything extra I want is as "vanilla" as possible.